core/assets/sounds/cues/animalAcrobat/data.json

```diff
@@ -0,0 +1,163 @@
+{
+  "gameID": "animalAcrobat",
+  "gameName": "Animal Acrobat",
+  "series": "megamix",
+  "cues": [
+    {
+      "id": "animalAcrobat/Animal Acrobat- Beginning leap",
```

Comment:
Rename the files so that they are in lowerCamelCase. Also, please omit the "Animal Acrobat- " part.